Makefile

```diff
@@ -700,6 +700,9 @@ endif
 zynq_runtime: $(ROOT_DIR)/apps/hls_examples/hls_support/HalideRuntimeZynq.cpp $(INCLUDE_DIR)/HalideRuntime.h
 	$(CXX) -c -Wall $(ROOT_DIR)/apps/hls_examples/hls_support/HalideRuntimeZynq.cpp -o $(BUILD_DIR)/HalideRuntimeZynq.o -I$(INCLUDE_DIR)
 	ar rvs $(LIB_DIR)/libZynqRuntime.a $(BUILD_DIR)/HalideRuntimeZynq.o
+	# test zynq codegen
+	make -C apps/hls_examples/unsharp_hls/ run_zynq	
```

Comment:
Could you add comments about the difference of what is tested in two folders? I think you could move some text from the PR description.